metron-platform/metron-parsers/src/main/java/org/apache/metron/parsers/interfaces/MessageParser.java

```diff
@@ -34,15 +34,15 @@
    * @param rawMessage
    * @return If null is returned, this is treated as an empty list.
    */
-  List<T> parse(byte[] rawMessage);
+  List<T> parse(byte[] rawMessage, SensorParserConfig sensorParserConfig);
```

Comment:
If we only want to check for changes every N minutes, instead of on every message, this approach gives us a central location to implement that policy in ParserBolt too, without every parser having to worry about it.